lib/components/molecules/QuoteMedia/types/index.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import type { Node } from 'react';
+
+export type Props = {
+  className?: string,
+  title?: string,
+  author: string,
```

Comment:
author: string,
  quoteContentProp: string,
  quoteClassName?: string, 

are not needed as part of this component.